Client/Frontend/Browser/TabToolbar.swift

```diff
@@ -188,7 +188,7 @@ class TabToolbar: UIView {
         return ToolbarButton()
     }()
 
-    fileprivate let privateModeBadge = BadgeWithBackdrop(imageName: "privateModeBadge", backdropCircleColor: UIColor.Defaults.MobilePrivatePurple)
+    fileprivate let privateModeBadge = BadgeWithBackdrop(imageName: "privateModeBadge", backdropCircleColor: UIColor.MobilePrivatePurple)
```

Comment:
This code looks dead to me. Can we investigate if it shouldn't be removed?
(we don't have hideImages feature anymore either)